ai-library/base/kustomization.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
+- crd.yaml
+- service-account.yaml
+- role.yaml
+- role-binding.yaml
+- operator.yaml
```

Comment:
@crobby  Does the order here matter? is it bottom up? or top down.?

Reply:
I think it's top-> down.